packages/datastore/datastore-codegen/tests/__snapshots__/Cli.test.ts.snap

```diff
@@ -0,0 +1,107 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`generate 1`] = `
+"export interface Note {
+    id: string;
+    title: string;
+    description: string;
+    comments: string[]
+    _version?: number;
+    _deleted?: boolean;
+}
+
+export type NoteCreate = Omit<Note, \\"id\\">;
+export type NoteChange =  Pick<Note, \\"id\\"> && Partial<NoteCreate>;
+
+export interface Comment {
+    id: string;
+    title: string;
+    description: string;
+    noteComment: string
+    _version?: number;
+    _deleted?: boolean;
+}
+
+export type CommentCreate = Omit<Comment, \\"id\\">;
+export type CommentChange =  Pick<Comment, \\"id\\"> && Partial<CommentCreate>;
+"
+`;
+
+exports[`generate 2`] = `
+"{
+  \\"Note\\": {
+    \\"name\\": \\"Note\\",
+    \\"version\\": 1,
+    \\"type\\": \\"object\\",
+    \\"primaryKey\\": \\"id\\",
+    \\"properties\\": {
+      \\"id\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"id\\",
+        \\"isRequired\\": true,
+        \\"index\\": true,
```

Comment:
The index on ID is redundant. The index is being used to apply extra indexes. ID do not need it.